lib/Interpreter/Interpreter.cpp

```diff
@@ -1034,6 +1037,21 @@ namespace cling {
     return name.startswith(utils::Synthesize::UniquePrefix);
   }
 
+  clang::SourceLocation Interpreter::getSourceLocation(bool skipWrapper) const {
+    const Transaction* T = getLatestTransaction();
+    if (!T)
+      return SourceLocation();
+
+    const SourceManager &SM = getCI()->getSourceManager();
+    if (skipWrapper) {
+      cling::ostrstream Strm;
+      Strm << "void " << utils::Synthesize::UniquePrefix << m_UniqueCounter
+           << "(void* vpClingValue) {\n ";
```

Comment:
This looks really fragile - if we ever change `WrapInput()` then we'll get a wrong offset here.

Could we move this code into a separate function and then have it called from here and from `WrapInput()`?